web/sites/default/config/facets.facet.group_document_liquor_application_neighborhood.yml

```diff
@@ -0,0 +1,61 @@
+uuid: 2f2dcf2e-b8de-462f-8ff5-9c68b9946c02
+langcode: en
+status: true
+dependencies:
+  config:
+    - search_api.index.database_full_index
+    - views.view.db_search_group_documents
+  module:
+    - search_api
+id: group_document_liquor_application_neighborhood
+name: 'Filter by Neighborhood'
+url_alias: neighborhood
+weight: -2
+min_count: 1
+show_only_one_result: false
+field_identifier: field_neighborhood_2
+facet_source_id: 'search_api:views_page__db_search_group_documents__liquor_applications'
+widget:
+  type: checkbox
+  config:
+    show_numbers: true
+    soft_limit: 0
+    soft_limit_settings:
+      show_less_label: 'Show less'
+      show_more_label: 'Show more'
+    show_reset_link: true
+    reset_text: 'Show all'
+    hide_reset_when_no_selection: true
+query_operator: and
+use_hierarchy: true
```

Comment:
QA did not test the use of a child neighborhood to make sure the hierarchy correctly displays. When I reviewed, I could not immediately get a child neighborhood to render the city section correctly. Perhaps it was because there was only one result for that neighborhood?

As a side note, the entity reference autocomplete field isn't ideal for selecting a neighborhood. It requires that the person entering the data know all the neighborhoods by exact name. We don't need to fix that in this PR, but I think I will make sure to add a select2 widget to this field when we roll out the feature to the editors in Civic Life.

Reply:
I'm not exactly sure what you mean by a child neighborhood, but perhaps this is something we can resolve when we refactor this view in the following story.

Reply:
I must mean that the terms in that vocabulary are arranged in a hierarchy. Editors will usually want to select the deepest term they can, which is usually a specific neighborhood. (Sextant > Coalition > Neighborhood) If you use the neighborhood facets on the construction view, you'll see what I mean.

It think it would be fine to resolve in that story.